src/pages/page-my-profile.html

```diff
@@ -565,6 +593,17 @@
                 }
                 this.path = null;
             }
+            /**
+            * blurs the current page
+            */
+            _blur() {
```

Comment:
can we use something like
```
          this.customStyle['--my-elem-visibility'] = 'visible';
          this.updateStyles();
```
instead of adding and removing classes